packages/icons/src/SVGBase/index.js

```diff
@@ -11,6 +11,7 @@ const SVG = ({
     y="0px"
     width="100%"
     height="100%"
+    className={className}
```

Comment:
className would already have been picked up and passed on by `...rest` so this change doesn't do anything, but perhaps good to have in for clarity anyway?